builder/builder.go

```diff
@@ -1737,13 +1738,37 @@ func (b *Builder) BuildImage(format string, template string) error {
 		format = b.State.Mix.Format
 	}
 
+	// swupd (client) called by itser will need a temporary directory to act as its stage dir.
+	wd, _ := os.Getwd()
+
 	// If the user did not pass in a template, default to release-image-config.json
 	if template == "" {
 		template = "release-image-config.json"
+
+		// If  the default image template is not present in the mix workspace,
+		// copy it from the default ister config directory.
+		templateFile := filepath.Join(wd, template)
+		if _, err := os.Stat(templateFile); os.IsNotExist(err) {
+			var isterConfigDir = "/usr/share/defaults/ister"
```

Comment:
This is a hard-coded variable. List it as a const at global scope. That way when (if) it changes for Clear Linux it is easier to find and changing it in one place changes it for the entire codebase. Really this is a configuration thing but let's not add config options right now.

Reply:
Done.